third_party/bigframes_vendored/pandas/core/generic.py

```diff
@@ -1126,7 +1139,8 @@ def pipe(
                 A dictionary of keyword arguments passed into ``func``.
 
         Returns:
-            same type as caller
+            T:
```

Comment:
Could we have this be `bigframes.pandas.DataFrame` or `bigframes.pandas.Series`, instead?